fdm-app/app/components/custom/sidebar-app.tsx

```diff
@@ -40,13 +41,13 @@ import {
     Sprout,
     Square,
 } from "lucide-react"
-
 import { Button } from "@/components/ui/button"
 import { useIsMobile } from "@/hooks/use-mobile"
 import { useEffect, useState } from "react"
 import { Form, NavLink } from "react-router"
 import { toast } from "sonner"
 import { useFarm } from "@/context/farm-context"
+import posthog from "posthog-js"
 
```

Comment:
you need full library?

Reply:
Yes, that is prefered way to load the package according to posthog